examples/src/main/java/com/google/instrumentation/trace/BasicTracing.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.instrumentation.trace;
+
+/** Example showing how to create a {@link Span} and add annotations. */
+public final class BasicTracing {
+  // Per class Tracer.
+  private static final Tracer tracer = Tracer.getTracer();
+
+  private static void doWork() {
+    try (Span span = tracer.startSpan(null, "MyRootSpan", StartSpanOptions.getDefault())) {
```

Comment:
so in this case, this is try-with closeable. In brave, I unimplemented closeable on span for some various reasons, though no-one else reviewed the change and @reta asked about this practice... we can consider reverting it.

https://github.com/openzipkin/brave/pull/352

what were your thoughts on having span be closeable? Assuming for simplicity or maybe because opentracing does it?